plugins/plugin.proto

```diff
@@ -84,18 +96,29 @@ message PluginResponse {
   // status code.
   repeated string error = 1;
   
-  // text output
+  // text output, typically written to stdout by openapic.
```

Comment:
I agree that we can drop it and let plugins just write to stdout or return files.